modules/proxistoreBidAdapter.js

```diff
@@ -0,0 +1,125 @@
+const { registerBidder } = require('../src/adapters/bidderFactory');
+const BIDDER_CODE = 'proxistore';
+
+function _getFormatSize(sizeArr) {
+  return {
+    width: sizeArr[0],
+    height: sizeArr[1]
+  }
+}
+
+function _createServerRequest(bidRequest, bidderRequest) {
+  const payload = {
+    bidId: bidRequest.bidId,
+    auctionId: bidRequest.auctionId,
+    transactionId: bidRequest.transactionId,
+    sizes: bidRequest.sizes.map(_getFormatSize),
+    website: bidRequest.params.website,
+    language: bidRequest.params.language,
+    gdpr: {
+      applies: false
+    }
+  };
+
+  const options = {
+    contentType: 'application/json',
+    withCredentials: true
+  };
+
+  if (bidderRequest && bidderRequest.gdprConsent) {
+    if ((typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') && bidderRequest.gdprConsent.gdprApplies) {
+      payload.gdpr.applies = true;
+    }
+    if ((typeof bidderRequest.gdprConsent.consentString === 'string') && bidderRequest.gdprConsent.consentString) {
+      payload.gdpr['consentString'] = bidderRequest.gdprConsent.consentString;
+    }
+  }
+
+  return {
+    method: 'POST',
+    url: bidRequest.params.url || '//abs.proxistore.com/' + bidRequest.params.language + '/v3/rtb/prebid',
+    data: JSON.stringify(payload),
+    options: options
+  };
+}
+
+function _createBidResponse(response) {
+  return {
+    requestId: response.requestId,
+    cpm: response.cpm,
+    width: response.width,
+    height: response.height,
+    ad: response.ad,
+    ttl: response.ttl,
+    creativeId: response.creativeId,
+    currency: response.currency,
+    netRevenue: response.netRevenue,
+    vastUrl: response.vastUrl,
+    vastXml: response.vastXml,
+    dealId: response.dealId
+  }
+}
+
+/**
+ * Determines whether or not the given bid request is valid.
+ *
+ * @param bid  The bid params to validate.
+ * @return boolean True if this is a valid bid, and false otherwise.
+ */
+function isBidRequestValid(bid) {
+  return !!(bid.params.website && bid.params.language);
```

Comment:
I would suggest maybe logging a warning / error here if the user does not have one of your required params.

Because the default console log does not point out exactly what the issue is!

![image](https://user-images.githubusercontent.com/13972794/68628714-596ad300-0496-11ea-9f39-6ebd5fea49ca.png)


Maybe something like:

```javascript
utils.logWarning('Proxistore Adapter: Both language and website are required params');
```

No big deal though you don't have to!